1_assignment/alphbet_frequency.js

```diff
@@ -1,13 +1,18 @@
 
-string = "u@#d$fkj$#!kds"
 
-function calculateFrequency(string) {
+//Sample String .
+var string = "u@#d$fkj$#!kds";
+
+
+
+// Solution 1 : Using String Functions.
+function calculateFrequencyUsingString(string) {
   // Write your code here
   var i=0;
   frequency=new Array(26).fill(0);
-  for (i =0; i< string.length ; i++)
+  for (i = 0; i < string.length ; i++)
   {
```

Comment:
Starting braces should be in the same line. Review comment not implemented nor here neither in below lines.